src/main/java/com/cs98/VerbatimBackend/request/CustomChallengeRequest.java

```diff
@@ -0,0 +1,14 @@
+package com.cs98.VerbatimBackend.request;
+
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class CustomChallengeRequest {
+    private String groupName;
```

Comment:
Can't uniquely identify a group just based on this info. One user might belong to multiple groups with the same name